src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -5,9 +5,12 @@ import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
 import createWrapper from '../../../lib/components/test-utils/selectors';
 import { scrollbarThickness } from '../../__integ__/scrollbars';
 import { viewports } from './constants';
-import { AppLayoutSplitViewPage } from './utils';
+import { AppLayoutSplitViewPage, getUrlParams } from './utils';
 
 import mobileStyles from '../../../lib/components/app-layout/mobile-toolbar/styles.selectors.js';
+import tableScrollbarStyles from '../../../lib/components/table/sticky-scrollbar/styles.selectors.js';
+
+const scrollbarSelector = `.${tableScrollbarStyles['sticky-scrollbar-visible']}`;
```

Comment:
That's how we access this element in other integration tests: https://github.com/cloudscape-design/components/blob/baad5f74be6dbca16b1c8e66b1aeab46709a7a6b/src/table/__integ__/sticky-scrollbar.test.ts#L10